lib/modules/home/home_view.dart

```diff
@@ -17,48 +43,26 @@ class HomeViewImpl extends StatefulWidget {
 
 class _HomeViewImplState
     extends ViewState<HomeViewImpl, HomeModule, HomeViewDelegate>
+    with AlertViewMixin, ProgressHUDViewMixin
     implements HomeView {
   final _dateTime = BehaviorSubject<DateTime>();
   final _user = BehaviorSubject<UserInfo>();
-  final _surveys = BehaviorSubject<List<SurveyInfo>>();
+  final _surveys = BehaviorSubject<List<SurveyInfo>>.seeded([]);
   final _pageController = CarouselController();
   final _currentPage = BehaviorSubject<int>.seeded(0);
   final _isLoading = BehaviorSubject<bool>.seeded(false);
```

Comment:
hmm, I see that you are using private vars outside `_HomeViewImplState` class by adding `part of 'file.dart'` like cheat 😂 The private variables should be only used in scope of a class (`_HomeViewImplState` in this case). If you wanna use it outside, please remove `_` prefix since this is incorrect, and we should not abuse `part of 'file.dart'`.
I notice that you apply this approach in many places 😂

Reply:
I think because `_HomeViewImplState` is private so declare private variables are kindly redundant there. Let's update them at the end, cause a lot of changes there 😢 
But for this point `The private variables should be only used in scope of a class` isn't quite right, for example

```dart
class A {
 final _title = "Hi"
}

// If class B is defined in same file or using `part`
class B {
 final title => A()._title + " there"
}

```
It's a good practice to hide` title` in A from the outside, but B still have the access 😁

Reply:
@markgravity yes, that's why I mentioned `part of 'file.dart'`, it's the same as your above example. 🤪 
And for `The private variables should be only used in scope of a class`, I think it's quite correct in other languages, such as Kotlin, Java, ... I have tried them, I don't understand why dart can get the private vars like that, maybe it's a feature 😂 

> I think because _HomeViewImplState is private so declare private variables are kindly redundant there. Let's update them at the end, cause a lot of changes there 😢

Yes I think so, we can create a PR to update all 👌